corehq/blobs/pillow.py

```diff
@@ -19,46 +14,28 @@
 
 class BlobDeletionProcessor(PillowProcessor):
 
-    def __init__(self, blob_db, bucket_base):
+    def __init__(self, blob_db):
         super(BlobDeletionProcessor, self).__init__()
         self.blob_db = blob_db
-        self.bucket_base = bucket_base
 
     def process_change(self, pillow_instance, change):
         if change.deleted:
-            bucket = join(self.bucket_base, change.id)
+            bucket = "{}/{}".format(change.meta.data_source_name, change.id)
```

Comment:
this is a bit of a mess (my fault). my intention was that by the time they get to the pillow soft and hard deletions could be treated the same, so the kafka feed populates `deleted` with the value from the `ChangeMeta.is_deleted` [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/change_feed/consumer/feed.py#L160).

I think the long term thing to do would be to get rid of one of them (and also maybe add a way to flag soft deletions if we do need to distinguish between them as you suggest). But for now I think this should work as long as we update the kafka feed to publish hard deletions and don't process deletions from forms.